src/views/repository/Translate.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <repository-view-base
+    :repository="repository"
+    :loading="loading"
+    :error-code="errorCode">
+    <div v-if="repository">
+      <div class="bh-notification">
+        <div
+          v-if="repository.authorization &&
```

Comment:
Check if user is not authenticated, after check if can cantribute in this Repository.